fire/cli/ts/gnss.py

```diff
@@ -234,68 +252,580 @@ def klargør_celle(input):
+    :math:`z\\lt Z_{1-\\alpha/2}`). Ellers forkastes nulhypotesen og den alternative
+    hypotese må accepteres.
+
+
+    **Pooled varians**
+
+    Den poolede varians :math:`\\sigma_{pooled}^2` beregnes som et vægtet gennemsnit af
+    den estimerede varians :math:`\\sigma_{0,j}^2` for hver tidsserie
+    :math:`j=1\\,..N_{ts}` som indgår i ensemblet:
+
+    .. math::
+        \\sigma_{pooled}^2 = \\frac{\\sum_{j=1}^{N_{ts}} \\text{dof}_j \\,
+        \\sigma_{0,j}^2}{\\sum_{j=1}^{N_{ts}} \\text{dof}_j} =
+        \\frac{\\sum_{j=1}^{N_{ts}} \\text{SSR}_j}{\\sum_{j=1}^{N_{ts}} \\text{dof}_j}
+
+    hvor :math:`\\text{dof}_j` er antallet af frihedsgrader for hver tidsserie :math:`j`.
+    Den poolede varians kan nu erstatte :math:`\\sigma_0^2` i de ovenstående formler,
+    hvorved de statistiske størrelsers "poolede" udgave opnås. Dette indebærer antagelsen
+    om, at :math:`\\sigma_{pooled}^2` er estimatet af *populationens* varians og ikke
+    *sample*-variansen som er :math:`\\sigma_{0}^2`. Dette gør os i stand til, i
+    hypotesetest og konfidensintervaller at anvende en normalfordeling med fraktilen
+    :math:`Z_{1-\\alpha/2}` som er uafhængig af antal frihedsgrader, i stedet for en
+    t-fordeling med fraktilen :math:`T_{1-\\alpha/2}`.
+
+    """
+    # denne funktion regner altid på 5D punkter. Kan evt. udvides.
+    tidsseriegruppe = "5D"
+    skalafaktor = 1e3
+
+    # Map parameter alias til rigtigt navn igennem GNSS_TS_PARAMETRE
+    try:
+        parameter = GNSS_TS_PARAMETRE[parameter]
+    except KeyError:
+        raise SystemExit(f"Ukendt tidsserieparameter '{parameter}'")
+
+    # Opret Tidsserieensemble
+    tsensemble = TidsserieEnsemble(
+        GNSSTidsserie,
+        min_antal_punkter=min_antal_punkter,
+        tidsseriegruppe=tidsseriegruppe,
+        referenceramme=referenceramme,
+    )
+
+    # Hent alle tidsserier i FIRE som passer til søgningen
+    query = fire.cli.firedb.session.query(GNSSTidsserie).filter(
+        GNSSTidsserie._registreringtil == None,
+        GNSSTidsserie.referenceramme == referenceramme,
+    )
+
+    # Filtrér på de givne tidsserienavne
+    if not alle:
+        if not ts_fil and not ts_liste:
+            raise SystemExit("Ingen tidsserier angivet.")
+
+        tidsserienavne = []
+
+        if ts_fil:
+            with open(ts_fil, "r") as f:
+                lines = list(f)
+            ts_fra_fil = [line.strip() for line in lines]
+            tidsserienavne += ts_fra_fil
+
+        if ts_liste:
+            ts_fra_liste = ts_liste.split(",")
+            tidsserienavne += ts_fra_liste
+
+        query = query.filter(GNSSTidsserie.navn.in_(tidsserienavne))
+
+    tidsserier = query.all()
+
+    # Filtrér desuden på tidsseriegruppe og min_antal_punkter, da de ikke kan filtreres via SQL
+    tidsserier = [
+        ts
+        for ts in tidsserier
+        if (
+            ts.tidsseriegruppe == tidsseriegruppe
+            and len(ts.koordinater) >= min_antal_punkter
+        )
```

Comment:
Véd ikke, om det er black, der formatter sådan, men det er ikek nødvendigt at have parenteser omkring, hvis man tilføjer flere linjer med betingelser.
```suggestion
        ts.tidsseriegruppe == tidsseriegruppe
        and len(ts.koordinater) >= min_antal_punkter
```

Reply:
Jep det er black der laver linjeombrydningen sådan. Den laver ikke parenteserne, og var ikke lige klar over at man kan undvære dem. Men jeg kan nu alligevel godt lide at det, synes bliver nemmere at overskue hvad der er med og ikke med i if statementet.